moveit_experimental/moveit_jog_arm/package.xml

```diff
@@ -27,6 +27,9 @@
 
   <exec_depend>joy_teleop</exec_depend>
 
+  <build_depend>message_generation</build_depend>
```

Comment:
Let's put the build_depend at line 21 and the exec_depend with the other exec_depend. I think they are usually sorted like that. So it will look like:

<build_depend>

<depend>
<depend>
...

<exec_depend>
<exec_depend>

Reply:
Done